docs/clients/dotnet/5.0/subscriptions/catchup-subscriptions.md

```diff
@@ -14,8 +14,51 @@ When the subscription starts for the first time and the stream it subscribes to
+
 ## Usage
 
 Catch-up subscriptions are typically used for producing _read models_ in event-sourced systems that use the CQRS pattern. Subscribers that update read models are often called _projections_ because they project the event payload to a piece of state in another database. Client-side projections use the same concept as EventStoreDB server-side projections but have a different purpose.
 
+::: tip Storing checkpoints
+The best practice for subscriptions that project events to another storage, is to store checkpoints in the same storage. Projecting an event and storing the checkpoint in one transaction allows you t achieve the _exactly once_ event processing.
+:::
+
+## Subscribing to a stream
+
+You can subscribe to any individual event stream in EventStoreDB. It could be a normal stream, where your software append events, or a stream produced by the server-side projection, either a system projection (like `$et-SomethingHappened`) or a custom projection. 
+
+Use the `IEventStoreConnection.SubscribeToStreamFrom` method to initiate the subscription. The connection must be open by the time you call this method.
+
+You need to specify the stream, which you want to subscribe to, the last known checkpoint, subscription settings and the event handling function. Optionally, you can a function, which then gets called when the subscription switches from processing historical events to real-time processing, and a function for handling subscription drops.
+
+::: tip Dropping subscription
+There are multiple reasons for a subscription to drop. The connection might close due to network issues, the subscription might get overloaded with events, or your event handler will throw an unhandled exception. It is usually a good idea to handle subscription drops and resubscribe when needed, to overcome transient issues. When a subscription drops, the application would keep working but will not process any events.
+:::
+
+<<< @/docs/clients/dotnet/5.0/sample-code/Subscriptions/CatchUp.cs#SubscribeToStream
+
+In this code, we create an instance of `CatchUpSubscriptionSettings`. You can also use `CatchUpSubscriptionSettings.Default` with default settings instead.
+
+## Subscribing to `$all`
+
+Subscribing to the global event stream gives you a possibility to create read models from many different event streams. It is a powerful method to create, for example, reporting models with aggregated and denormalized data without using common database operations like `JOIN`. You must, however, carefully evaluate your subscription performance, as when you subscribe to `$all`, you'll get absolutely everything what gets written to the EventStoreDB cluster. You might also need to filter out system events, by checking if event type starts with `$`. In normal applications, you won't need to process system events.
```

Comment:
We don't have serverside filtering documented for 5.0 do we? Because we could mention that you can filter system events using that too?

Reply:
Can't find anything in the API. If you remember where is it, I can add it. I was looking for it yesterday, using 5.0.9 client, but found nothing.